oak/server/wasm_node.cc

```diff
@@ -561,4 +600,14 @@ wabt::interp::HostFunc::Callback WasmNode::OakRandomGet(wabt::interp::Environmen
   };
 }
 
+wabt::interp::HostFunc::Callback WasmNode::WasiPlaceholder() {
```

Comment:
Add a comment here that clarifies that this function will always log and return an error (and terminate the program I guess, since it's logging as `fatal`)

Reply:
Done in the header file